exchanges/gateio/gateio_wrapper.go

```diff
@@ -1849,33 +1855,27 @@ func (g *Gateio) GetHistoricCandlesExtended(ctx context.Context, pair currency.P
 	if err != nil {
```

Comment:
remove `validate kline` and `formateExchangeCurrency` as they are done on the GetKlineRequest calls